README.md

```diff
@@ -162,6 +174,116 @@ string, default: `my-cluster`
 
 Name of the cluster.
 
+#### `ha_cluster_resource_primitives`
+
+Structure, default: no resources
+
+```yaml
+ha_cluster_resource_primitives:
+  - id: "resource-id"
+    agent: "resource-agent"
+    instance_attrs:
+      - attrs:
+          attribute1_name: "attribute1_value"
+          attribute2_name: "attribute2_value"
+    meta_attrs:
+      - attrs:
+          meta_attribute1_name: "meta_attribute1_value"
+          meta_attribute2_name: "meta_attribute2_value"
+    operations:
+      - action: "operation1-action"
+        attrs:
+          operation1_attribute1_name: "operation1_attribute1_value"
+          operation1_attribute2_name: "operation1_attribute2_value"
+      - action: "operation2-action"
+        attrs:
+          operation2_attribute1_name: "operation2_attribute1_value"
+          operation2_attribute2_name: "operation2_attribute2_value"
+```
+
+This variable defines pacemaker resources (including stonith) configured by the
+role. The items are as follows:
+
+* `id` (mandatory) -  Id of a resource.
+* `agent` (mandatory) - Name of a resource or stonith agent, for example
+  `ocf:pacemaker:Dummy` or `stonith:fence_xvm`. It is mandatory to use
+  `stonith:` for stonith agents. For resource agents, it is possible to use a
+  short name, such as `Dummy` instead of `ocf:pacemaker:Dummy`. However, if
+  several agents with the same short name are installed, the role will fail as
+  it will be unable to decide which agent should be used. Therefore, it is
+  recommended to use full names.
+* `instance_attrs` (optional) - List of sets of the resource's instance
+  attributes. Currently, only one set is supported. The exact names and values
+  of attributes, as well as whether they are mandatory or not, depends on the
+  resource or stonith agent.
+* `meta_attrs` (optional) - List of sets of the resource's meta attributes.
+  Currently, only one set is supported.
+* `operations` (optional) - List of the resource's operations.
+  * `action` (mandatory) - Operation action as defined by pacemaker and the
+    resource or stonith agent.
+  * `attrs` (mandatory) - Operation options, at least one option must be
```

Comment:
Are the names of the attributes guaranteed to be valid Python identifiers?  That is, can you have something like
"@attr-name" or some other name that isn't a valid Python identifier?  If so, then you won't be able to use `attribute1_name: value` - you'll have to use a `list` of `name: attr_name` `value: attr_value` pairs like this:
```yaml

    instance_attrs:
      - attrs:
          - name: attribute1_name
            value: "attribute1_value"
          - name: attribute2_name
            value: "attribute2_value"
```

Reply:
No, that cannot be guaranteed. I switched to the format you proposed. Thanks.